spec.bs

```diff
@@ -4395,6 +4427,16 @@ Add the following algorithm:
      null, and return null otherwise.
 </div>
 
+<div algorithm=network-partition-key-patch>
+  Rewrite the [=network partition key=] redefinition to read:
+
+  A [=network partition key=] is a [=tuple=] consisting of:
+
+  - a [=site=].
+  - null or an [=implementation-defined=] value.
+  - A <dfn for="network partition key">nonce</dfn> (null or an [=implementation-defined=] value).
```

Comment:
Be consistent with capitals in this list. Also: are there other specs that monkeypatch the same definition to add a nonce? If so, we should name this "fenced frame nonce". Or maybe we should rename this regardless, to future-proof it.

Reply:
I'll make it lowercase to match the convention we use in the fenced frame spec and to match the case of "null".

The [iframe credentialless spec](https://wicg.github.io/anonymous-iframe/#spec-network-partition-key) introduces and uses this nonce, but none of the items in its network partition key definition are named. I'm just introducing a name to make it easier to get in the spec. Really this is a fenced frame or iframe credentialless nonce. Alternatively, I could also keep it unnamed, but then I'd need to call `|network partition key|[2]` to get access to it, but that seems harder to read.

If we want to name all 3 fields, it would probably be `site`/`top level site`, `second key`, and `nonce`/`credentialless or fenced frame nonce` respectively.

Let me know what you'd prefer.

Reply:
Explicit naming sounds good! Let's just link to https://wicg.github.io/anonymous-iframe/#spec-network-partition-key from "redefinition" from clarity, just with a normal anchor tag.

Reply:
Kept the explicit naming with the nonce, and added a link.